OBSDOCS-1211-Logging 5.8.10 release notes (#79100)
openshift/openshift-docs: modules/logging-release-notes-5-8-10.adoc

// module included in /logging/logging-5-8-release-notes
:_mod-docs-content-type: REFERENCE
[id="logging-release-notes-5-8-10_{context}"]
= Logging 5.8.10
This release includes link:https://access.redhat.com/errata/RHBA-2024:4629[OpenShift Logging Bug Fix Release 5.8.10] and link:https://access.redhat.com/errata/RHBA-2024:4665[OpenShift Logging Bug Fix Release 5.8.10].

[id="logging-release-notes-5-8-10-known-issues"]
== Known issues

> * Before this update, while enabling retention, the Loki Operator produced an invalid configuration. As a result, Loki did not start up properly. With this update, Loki pods work fine with the ability to set retention. (link:https://issues.redhat.com/browse/LOG-5821[LOG-5821])

Reviewer: ```suggestion
* Before this update, when enabling retention, the Loki Operator produced an invalid configuration. As a result, Loki did not start properly. With this update, Loki pods can set retention. (link:https://issues.redhat.com/browse/LOG-5821[LOG-5821])
```
Just "start" https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s 
The rest for minimalism

[id="logging-release-notes-5-8-10-bug-fixes"]
== Bug fixes

> * Before this update, the `ClusterLogForwarder` introduced an extra space in the message payload which did not follow the `RFC3164` specification. With this update, the extra space has been removed, fixing the issue. (link:https://issues.redhat.com/browse/LOG-5647[LOG-5647])

Reviewer: ```suggestion
* Before this update, the `ClusterLogForwarder` introduced an extra space in the message payload that did not follow the `RFC3164` specification. With this update, the extra space has been removed, fixing the issue. (link:https://issues.redhat.com/browse/LOG-5647[LOG-5647])
```
"which" requires a comma and is used with non-restrictive clauses (meaning you can take that info out)
"that" does not take the comma and is used with restrictive clauses, meaning that the words after "that" are essential to understanding the sentence
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#w (which)
